doc/man3/SSL_shutdown.pod

```diff
@@ -108,6 +102,11 @@ nothing is to be done, but select() can be used to check for the required
 condition. When using a buffering BIO, like a BIO pair, data must be written
 into or retrieved out of the BIO before being able to continue.
 
+After SSL_shutdown() returned 0, it's possible to call SSL_shutdown() again
+to wait for the peer's "close notify" alert.
```

Comment:
What about @kaduk's proposal to use symbolic names instead of quoted strings, e.g. `close_notify` instead of `"close notifiy"`?

Reply:
See https://github.com/openssl/openssl/pull/7188#discussion_r216873219 (outdated).

Reply:
I really see no need to use the same terminology as in the RFCs. It doesn't really seem to add something to me. But on the other hand, I don't like the quotes around it either.

Looking at what we have, we actually seem to use close_notify in SSL_alert_type_string, and "close notify" in all the other manpages. So I guess I'll just change it everywhere.

Reply:
(To be clear, I am way too close to the RFC to have a useful opinion on whether or not it's helpful to stick to the RFC terminology or not.  So I would not mind at all if the consensus went the other way.)